spidy/package.json

```diff
@@ -15,6 +15,7 @@
     "mui": "^0.0.1",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "react-responsive-carousel": "^3.2.23",
```

Comment:
install the react-responsive-carousel for the image slideshow (animated) purpose